coc/client.py

```diff
@@ -278,13 +283,10 @@ def login_with_keys(self, *keys: str) -> None:
 
         LOG.debug("HTTP connection created. Client is ready for use.")
 
-    def close(self) -> None:
-        """Closes the HTTP connection
-        """
-        LOG.info("Clash of Clans client logging out...")
-        self.dispatch("on_client_close")
-        self.loop.run_until_complete(self.http.close())
-        self.loop.close()
+    async def close(self) -> None:
```

Comment:
Idk if anyone used it but should this still dispatch that `on_client_close` event?

Reply:
I tried looking at the dispatch stuff but I don't really understand it. Could you provide some guidance on it?